app/api/helpers/export_helpers.py

```diff
@@ -207,7 +208,15 @@ def export_event_json(event_id, settings):
     fp.close()
     # make zip
     shutil.make_archive(dir_path, 'zip', dir_path)
-    return dir_path + '.zip'
+    dir_path = dir_path + ".zip"
+
+    storage_path = UPLOAD_PATHS['exports']['zip'].format(
+        event_id = event_id
+    )
+    uploaded_file = UploadedFile(dir_path, dir_path.rsplit('/', 1)[1])
+    storage_url = upload( uploaded_file, storage_path )
```

Comment:
PR looks good functionality-wise. Neat work !!
One thing I see here is that you are giving extra trailing spaces in function parameters. eg `upload( uploaded_file, storage_path )`. Please remove them. 
We would like to follow a consistent style everywhere in the code.